.github/pull_request_template.md

```diff
@@ -19,9 +19,13 @@ Add a brief summary of your PR here, between one sentence and two paragraphs
 
 By placing an x in the boxes I certify that I have:
 
-- [ ] Ran `cargo test --all` locally if this modified any rs files.
-- [ ] Ran `cargo +stable fmt --all` locally if this modified any rs files.
 - [ ] Updated the content of the book if this PR would make the book outdated.
 - [ ] Added a changelog entry if this will impact users, or modified more than 5 lines of Rust that wasn't a doc comment.
-- [ ] Added unit tests for new APIs if any were added in this PR.
+- [ ] Added unit tests for new code added in this PR.
 - [ ] Acknowledged that by making this pull request I release this code under an MIT/Apache 2.0 dual licensing scheme.
+
+If this modified or created any rs files:
+
+- [ ] Ran `cargo +stable fmt --all`
```

Comment:
Do we really need to ask this? I feel like this doesn't add much. We check it on CI anyway.

Reply:
I'd favour asking it over not -- if someone pushes and forgets about it, then there's some impedence to getting the PR merged (yes I know, we can pull and push onto another fork).